compiler/src/iree/compiler/DispatchCreation/Passes.td

```diff
@@ -246,7 +246,9 @@ def FormDispatchRegionsPass :
     Option<"fusePadWithConsumers", "fuse-pad-with-consumers", "bool",
            /*default=*/"false", "Enable fusing pad with consumer">,
     Option<"fusePadWithProducers", "fuse-pad-with-producers", "bool",
-           /*default=*/"false", "Enable fusion of pad with producers">
+           /*default=*/"false", "Enable fusion of pad with producers">,
+    Option<"fuseTruncWithConsumers", "fuse-trunc-with-producers", "bool",
```

Comment:
I think you meant `fuseTruncWithProducers` here?

Reply:
The `Option` has c++ variable name of "fuseTruncWithConsumer" and the flag as `fuse-trunc-with-producers`. That seems like a typo